_content/tour/ind/struct-types.article

```diff
@@ -138,108 +138,108 @@ fields. This causes more padding inside the struct.
+Namun, jika ex2 diubah untuk dideklarasikan sebagai tipe struct tanpa nama menggunakan deklarasi yang sama persis
+seperti ex1, tidak ada sintaks konversi yang diperlukan.
 
     var ex2 struct {
         flag    bool
         counter int16
         pi      float32
     }
 
-    ex1 = ex2  // Allowed, NO need for conversion syntax
+    ex1 = ex2  // Diperbolehkan, tidak perlu ada konversi
 
-The compiler will allow this assignment without the need for conversion.
+Kompiler akan mengizinkan assignment ini tanpa perlu konversi.
 
-** Notes
+** Catatan
 
-- We can use the struct literal form to initialize a value from a struct type.
-- The dot (.) operator allows us to access individual field values.
-- We can create anonymous structs.
+- Bentuk literal struct dapat digunakan untuk menginisialisasi value dari tipe struktur.
+- Operator titik (.) memungkinkan untuk mengakses value dari suatu atribut.
+- Kita dapat membuat tipe struct tanpa nama.
 
 ** Quotes
 
 "Implicit conversion of types is the Halloween special of coding. Whoever thought
 of them deserves their own special hell." - Martin Thompson
 
-** Extra Reading
+** Bacaan Ekstra
 
-- [[https://www.ardanlabs.com/blog/2013/07/understanding-type-in-go.html][Understanding Type in Go]] - William Kennedy    
-- [[https://www.ardanlabs.com/blog/2013/07/object-oriented-programming-in-go.html][Object Oriented Programming in Go]] - William Kennedy    
-- [[https://dave.cheney.net/2015/10/09/padding-is-hard][Padding is hard]] - Dave Cheney    
-- [[https://www.geeksforgeeks.org/structure-member-alignment-padding-and-data-packing/][Structure Member Alignment, Padding and Data Packing]]    
-- [[http://www.catb.org/esr/structure-packing][The Lost Art of Structure Packing]] - Eric S. Raymond    
+- [[https://www.ardanlabs.com/blog/2013/07/understanding-type-in-go.html][Memahami tipe di Go]] - William Kennedy    
+- [[https://www.ardanlabs.com/blog/2013/07/object-oriented-programming-in-go.html][Pemrograman Berorientasi Objek di Go]] - William Kennedy    
+- [[https://dave.cheney.net/2015/10/09/padding-is-hard][Padding itu sulit]] - Dave Cheney    
+- [[https://www.geeksforgeeks.org/structure-member-alignment-padding-and-data-packing/][Alignment Struktur, Padding dan Pengemasan Data]]    
+- [[http://www.catb.org/esr/structure-packing][Seni yang Hilang tentang Pengemasan Struktur]] - Eric S. Raymond    
 
-* Exercises
+* Latihan
 
-Use the template as a starting point to complete the exercises. A possible solution is provided.
+Gunakan templat sebagai awal untuk menyelesaikan latihan. Solusi telah disediakan.
 
-** Exercise 1
+** Latihan 1
 
-*Part* *A:* Declare a struct type to maintain information about a user (name,
-email and age). Create a value of this type, initialize with values and display
-each field.
+*Bagian* *A:* Deklarasikan tipe struct untuk menyimpan informasi tentang pengguna (nama,
+email, dan usia). Buat value dari tipe ini, inisialisasi dengan value, dan tampilkan
```

Comment:
email should be translated to surel, value should be translated to nilai

Reply:
using `surel` is weird, I think email fit better since it's widely used instead of surel, and `nilai` isn't quite right for value, in my opinion `isi` is better for the context of this article, but not translating it, is making more sense because it sounds weird, since value is commonly used for people that already know programming.

@walvisk

Reply:
alright, i think `email` is also OK. but for `value` word i think we can multiple translation depend on context. maybe on this context `isi` is better, but for the other maybe `nilai` is more suitable

Reply:
yeah, because it's depend on the context, that's why I insist to keep `value` as is, not translated to bahasa @walvisk

Reply:
but still, i think `value` still need to be translated, mixed language may make reader confuse

Reply:
I assume people that read this, already have knowledge about programming and familiar with value, so it wont confuse them, wdyt @raufhm ?

Reply:
if you say it will not confuse them, then translate `value` to `nilai` will not confuse them. It also make our translation consistent. Value here refer to what inside it then `nilai` is also correct in my personal thinking. it is not a variable or technical term name that need to be kept as `value`.

Reply:
I guess you're correct